internal/credential/vault/private_library_test.go

```diff
@@ -1333,15 +1333,6 @@ func TestRepository_sshCertIssuingCredentialLibrary_retrieveCredential(t *testin
 			vaulthPath: "ssh/issue/boundary",
 			opts:       []Option{WithKeyType(KeyTypeEcdsa), WithKeyBits(384), WithExtensions("{ \"permit-port-forwarding\": \"\" }")},
 		},
-		{
-			name:     "vault sign boundary ec(224) invalid key",
-			username: "username-11-lucky-number",
-			expected: map[string]string{
-				"error": "invalid KeyBits. when KeyType=ecdsa, KeyBits must be one of: 256, 384, or 521",
-			},
-			vaulthPath: "ssh/sign/boundary",
-			opts:       []Option{WithKeyType(KeyTypeEcdsa), WithKeyBits(224)},
-		},
```

Comment:
This test was removed since the setup is no longer possible, the schema changes in this PR make it so a library can not be created with `224` for key bits.

Reply:
the setup is technically not possible, but the validation still exists in the function, and this test still correctly checks that the validation works. imo this test should not be removed

Reply:
Well, this test was failing during the setup phase due to it not being possible. I suppose we could implement the setup another way...

Reply:
oh gotcha, you're right those tests work by going through the db which wouldn't be possible with your changes. I think it's ok in that case. important part is that all code paths from db are checked